src/components/svgs/nodes/master.tsx

```diff
@@ -0,0 +1,298 @@
+        return svgs.map(s => {
+            const coreProps =
+                attrs.nodeType === 'Station' && attrs.core && attrs.core === s.id
+                    ? { id: `stn_core_${id}`, onPointerDown, onPointerMove, onPointerUp, style: { cursor: 'move' } }
+                    : {};
+            const calcAttrs = modifyAttributes(
+                s.attrs,
+                attrs.components.map(s => s.value),
+                attrs.components.map(s => s.type)
+            );
+            return (
+                <g key={s.id} transform={`translate(${calcAttrs.x ?? 0}, ${calcAttrs.y ?? 0})`}>
+                    {React.createElement(
+                        s.type,
+                        {
+                            ...calcAttrs,
+                            x: 0,
+                            y: 0,
+                            ...coreProps,
+                        },
+                        s.children
+                            ? dfsCreateElement(s.children)
+                            : '_rmp_children_text' in calcAttrs
+                              ? (calcAttrs._rmp_children_text as string)
+                              : null
+                    )}
+                </g>
+            );
+        });
+    };
+
+    const masterTransform = attrs.transform ?? defaultMasterTransform;
+
+    return React.createElement(
+        'g',
+        { id: id, transform: `translate(${x}, ${y})`, ...gPointerEvents },
+        attrs.randomId !== 'undefined' ? (
+            <g
+                transform={`translate(${masterTransform.translateX}, ${masterTransform.translateY}) scale(${masterTransform.scale}) rotate(${masterTransform.rotate})`}
+            >
+                {dfsCreateElement(attrs.svgs)}
+            </g>
+        ) : (
+            <>
+                <circle r={5} />
+                <text x={-3.75} y={3} fill="white" fontSize="8" fontWeight="bold">
+                    M
+                </text>
+            </>
+        )
+    );
+};
+
+export interface MasterAttributes extends MasterParam {}
+
+const defaultMasterAttributes: MasterAttributes = {
+    randomId: 'undefined',
+    label: 'Undefined',
+    transform: defaultMasterTransform,
+    nodeType: 'MiscNode',
+    svgs: [],
+    components: [],
+};
+
+const attrsComponent = (props: AttrsProps<MasterAttributes>) => {
+    const { id, attrs, handleAttrsUpdate } = props;
+    const dispatch = useRootDispatch();
+    const toast = useToast();
+    const {
+        paletteAppClip: { output },
+    } = useRootSelector(state => state.runtime);
+    const { t } = useTranslation();
+    const [openImport, setOpenImport] = React.useState(false);
+    const [openManager, setOpenManager] = React.useState(false);
+
+    const getComponentValue = (query: string) => {
+        attrs.components.forEach(c => {
+            if (c.id === query) {
+                return c.value ?? c.defaultValue;
+            }
+        });
+        return undefined;
```

Comment:
This is redundant?

Reply:
not found in attrs.components, so we should put the value from the configuration.

Reply:
when replacing a param, we copy the variables' data to new param.

(e.g. we have a master station `name = 'ABC'`. Then I replace the param with new one. The station name will be reserved while the replacement.)

If the variable that isn't in the old param, we should put new default value to it.

Reply:
But this function will always return nothing even without this line...

`const getComponentValue: (query: string) => undefined`

I mean the forEach iteration won't change anything and thus getComponentValue will always return undefined.